index.html

```diff
@@ -382,37 +381,54 @@
 
         </div> <!-- End of .container inside the main-wrapper section -->
 
-        <div class="social-media" id="social-media" href="#social-media">
+         <div class="social-media" id="social-media" href="#social-media">
           <div class="container-fluid">
             <!-- Start ignoring BootLintBear -->
             <div class="row">
                 <h2 class="white-header">FOSSASIA on social media</h2>
-                <div class="col-xs-12 col-md-6 facebook-widget"></div>
-                <div class="col-xs-12 col-md-6 g-page-embed">
-                    <div class="g-page" data-href="//plus.google.com/+fossasiaorg" data-width="370"></div>
-                </div>
             </div>
             <!-- Stop ignoring -->
             <!-- Start ignoring BootLintBear -->
             <div class="row">
-                <div class="col-xs-12 col-md-6">
-                    <a class="twitter-timeline" data-width="370" data-height="500"
-                    href="//twitter.com/fossasia">Tweets by fossasia</a>
+                <div class="col-xs-6 col-md-4">
+                    <div class="t-page">
+                    <div class="foss-twitter">
+                       <img src="images/twitter.png" alt="twitter" class="header-image" style="float:center" height=200 width=200>
+                        <a href="https://twitter.com/fossasia?lang=en" target="_blank"><h2>We're on Twitter! Follow us</h2></a>
+                    </div>
+                    </div>
+                </div>
+                <div class="col-xs-6 col-md-4 g-page-embed">
+                    <div class="g-page">
+                    <div class="google-foss">
+                        <img src="images/Google%20Plus%20Logo%20(4).png" alt="google-plus" class="header-image" style="float:center" height=200 width="200">
```

Comment:
The file name is absurd. Use a glyphicon/font-awesome icon instead. We load too many images as-is.